src/MainPage/MainPage.css

```diff
@@ -8,124 +8,85 @@
   display: flex;
   flex-direction: column;
   background-color:rgb(0, 0, 0);
-  padding: 2em;
-}
-
-.help-button {
-  border:white;
-  border-radius: 100%;
-  border-style:ridge;
-  border-width: .3em;
-  background-color: rgb(255, 0, 0);
-}
-
-.help-button:hover {
-  cursor:pointer;
-  transform: scale(1.05);
-  transition: all 0.1s ease-in-out;
+  width: 100%;
+  height: 100%;
+  padding: 1vw;
 }
 
 .play-container {
-  display: relative;
-  flex-direction: column;
-  min-width: 87vmin;
-  overflow: auto;
-  /* max-width: 100%; */
-  max-height: 100%;
-  margin: 1em;
-}
-.play-container
-.play-area-0,
-.play-area-1,
-.play-area-2,
-.play-area-3,
-.play-area-4,
-.play-area-5,
-.play-area-6,
-.play-area-7,
-.play-area-8,
-.play-area-9,
-.play-area-10,
-.play-area-11,
-.play-area-12
-{
-  background-repeat: no-repeat;
-  background-position: contain;
-  background-repeat: no-repeat;
-  min-height: 75vmin;
-  background-size: contain;
   display: flex;
-  flex-direction: row;
-  justify-content: center;
-  align-content: space-between;
-  border-style: ridge;
-  color: #000000;
-  border-color:rgb(255, 255, 255);
-  border-width: 1em;
+  flex-direction: column;
+  justify-content: space-between;
+  background-position: center;
+  background-size: cover;
+  border: 1em ridge rgb(255, 255, 255);
   border-radius: 3%;
+  width: 80vw;
+  height: 75vh;
+  overflow: hidden;
+  position: relative;
+  padding: 1vh;
 }
 
-.play-container .play-area-1 {
-  background-image: url('../Utilities/Images/Box_Beach_BDSP.png');
-}
-
-.play-container .play-area-2 {
-  background-image: url('../Utilities/Images/Box_Cave_BDSP.png');
-}
-
-.play-container .play-area-3 {
-  background-image: url('../Utilities/Images/Box_Checks_BDSP.png');
-}
-
-.play-container .play-area-4 {
-  background-image: url('../Utilities/Images/Box_City_BDSP.png');
+.ui-info {
+  display: flex; 
+  flex-direction: row; 
+  justify-content: space-between; 
+  width: 100%;
 }
 
-.play-container .play-area-5 {
-  background-image: url('../Utilities/Images/Box_Crag_BDSP.png');
-}
-
-.play-container .play-area-6 {
-  background-image: url('../Utilities/Images/Box_Desert_BDSP.png');
-}
-
-.play-container .play-area-7 {
-  background-image: url('../Utilities/Images/Box_Forest_BDSP.png');
-}
-
-.play-container .play-area-8 {
-  background-image: url('../Utilities/Images/Box_River_BDSP.png');
-}
-
-.play-container .play-area-9 {
-  background-image: url('../Utilities/Images/Box_Savanna_BDSP.png');
+.HUD {
```

Comment:
Glad we got this component where we want it on the main page, I believe al lot of the HUD work was done by Karl, but is not reflected in this PR due to merge conflicts in deleted branch.